package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "root",
+  "version": "0.59.0-rc2",
```

Comment:
Forgot to ask in the earlier PR, did you make sure the .bump-version script will bump this version here?

Reply:
This should get updated with the next RC build.  It's needed here to run the acceptance tests.